dbms/src/DataTypes/DataTypeTuple.cpp

```diff
@@ -166,6 +166,9 @@ void DataTypeTuple::serializeText(const IColumn & column, size_t row_num, WriteB
             writeChar(',', ostr);
         elems[i]->serializeAsTextQuoted(extractElementColumn(column, i), row_num, ostr, settings);
     }
+    // Special format for one element tuple (1,)
+    if (1 == elems.size())
+        writeChar(',', ostr);
```

Comment:
May be it's better not to add comma at all? We don't need to distinguish one element tuple from an expression in round brackets here, so tuples like `(1)` look fine. And it will not break tests and backward-compatibility. However, it may make sense to use new way formatting for `tuple(...)` function in SQL query (see `IAST::format(...)`), but I'm not sure. @alexey-milovidov, what do you think?